utils/go.mod

```diff
@@ -3,15 +3,89 @@ module github.com/openclarity/vmclarity/utils
 go 1.22.4
 
 require (
+	github.com/anchore/stereoscope v0.0.2
 	github.com/go-viper/mapstructure/v2 v2.0.0
 	github.com/moby/sys/mountinfo v0.7.1
 	github.com/onsi/gomega v1.33.1
+	github.com/openclarity/vmclarity/core v0.7.1
```

Comment:
should use replace directive in all places that import vmclarity packages, unrelated if indirect or direct import